use keystore::error::Error;

fn main() -> Result<(), Error> {
    keystore::run()
}
